tests/Unit/NumericalAlgorithms/LinearSolver/ConjugateGradient/Test_ConjugateGradientAlgorithm.input

```diff
@@ -0,0 +1,7 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: Test_ConjugateGradientAlgorithm
```

Comment:
remove these two lines, they have no effect as they are only parsed for input files in `tests/InputFiles`